docs/DeveloperGuide.adoc

```diff
@@ -866,6 +866,24 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |`* *` |teacher |list all grade components |view all the grade components that I have added to my module
 
 |`* * *` |teacher |assign grades of students |keep track of the grades that I have given out to students
+
+|`* * *` |teacher |create a class|assign students to the class
+
+|`* * *` |teacher |delete a class|remove a class that is created wrongly
+
+|`* * *` |teacher |assign student to class|add students to a class in the event that some students still have not signed up for a slot when classes begin
+
+|`* * *` |teacher |unassign student from class|remove a student from a class to prevent clashes in his/her timetable, or the student has dropped out of school, or if he/she has not paid his/her school fees
+
+|`* * *` |teacher |modify class enrollment limit|set class enrollment limits so that the classes that I’m teaching or my TAs are teaching are not over-subscribed.
+
+|`* * *` |teacher |access class attendance list|access and view the class attendance to see which students are present/absent
+
+|`* * *` |teacher |mark class attendance list|mark the attendance for every present student
+
+|`* * *` |teacher |mModify class attendance list|alter a wrongly-marked attendance for a specific student
```

Comment:
typo in `mModify`

Reply:
it would also be nice to add a space after the action and before the `|` denoting the next column